### PR TITLE
Update the version, and add back the universal links changes

### PR DESCRIPTION
**Pull request by orta**
I [compared 4.0.2 to master](https://github.com/artsy/eigen/compare/4.0.2-0...master) and spotted that a bad rebase had removed some of the code for universal links. This brings that back, and prepares us for making a 4.0.3 build.

### CODE REVIEW

**Comment by alloy** [comment]
I’m a little unclear on what rebase this is about, do you know what PR caused this? And t be sure, this is about code that changed _after_ 4.0.2 was released, right?

**Comment by alloy** [comment]
Ironically needs a rebase btw ;)

**Comment by orta** [comment]
Looks like it sneaked in with https://github.com/artsy/eigen/pull/2549

File: CHANGELOG.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 upcoming:
-    version: 4.0.2
+    version: 4.0.3
     emission_version: 1.4.6
     dev:
       - Fixes universal links in iOS 11 - sarah
```
Comment (alloy) [inline CHANGELOG.yml]: This is in 4.0.2

Comment (orta) [reply CHANGELOG.yml]: 👍

**Pull request merged**